[22.03] kernel: bump 5.10 to 5.10.168

hauke opened the pull request:
All patches automatically rebased.

Build system: x86_64
Build-tested: ramips/tplink_archer-a6-v3
Run-tested: ramips/tplink_archer-a6-v3

Signed-off-by: John Audia <therealgraysky@proton.me>
(cherry picked from commit 712681458a65736d3fc625bb9c481c31b23c1f97)

graysky2 committed:
kernel: bump 5.10 to 5.10.167

All patches automatically rebased.

Build system: x86_64
Build-tested: ramips/tplink_archer-a6-v3
Run-tested: ramips/tplink_archer-a6-v3

Signed-off-by: John Audia <therealgraysky@proton.me>
(cherry picked from commit 712681458a65736d3fc625bb9c481c31b23c1f97)

graysky2 commented:
@hauke - I have a PR for 5.10.168 but since you own this one, can you cherry-pick it from my branch?

graysky2 committed:
kernel: bump 5.10 to 5.10.168

Manually rebased:
  backport-5.10/804-v5.14-0001-nvmem-core-allow-specifying-of_node.patch

Removed upstreamed:
  generic-backport/807-v5.17-0003-nvmem-core-Fix-a-conflict-between-MTD-and-NVMEM-on-w.patch[1]

All other patches automatically rebased.

1. https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/?h=v5.10.168&id=34ec4c7831c416ac56619477f1701986634a7efc

Build system: x86_64
Build-tested: ramips/tplink_archer-a6-v3
Run-tested: ramips/tplink_archer-a6-v3

Signed-off-by: John Audia <therealgraysky@proton.me>
(cherry picked from commit 461072fc7b3d8fa77347a884fe5d36c81f660da8)
[Refresh on OpenWrt 22.03]
Signed-off-by: Hauke Mehrtens <hauke@hauke-m.de>

Borromini commented:
Compile-tested: ath79/generic, ramips/{mt7620,mt7621,mt76x8}, realtek/{rtl838x,rtl930x}, x86/64.
Run-tested: mt7621, rtl838x.
